src/pages/landingPage.js

```diff
@@ -19,37 +16,75 @@ import { Fragment } from 'react';
 
 
 function LandingPage() {
-  const start=()=>{
-    window.location.hash='/report-repairs'
-  }
+  const contactDetailsPageLink = (
+    <Link href="https://www.lincoln.gov.uk/contact-1/get-touch">
```

Comment:
In the [prototype](https://southwark-repairs-alpha-master.herokuapp.com/beta/v2/) clicking the "contact details page" link opens the url in a new page.
```suggestion
    <Link href="https://www.lincoln.gov.uk/contact-1/get-touch" target="_blank">
```